### PR TITLE
feature(code-export): add support to grapqhl

### PR DESCRIPTION
**Pull request by ajubin**
# Description

Add support to export GraphQL request into code to complete this issue, more specifically the explanation of the PR is on the issue comment linked down below

https://github.com/usebruno/bruno/issues/1031#issuecomment-1870248832

### What did I do ?

- [x] Enable dropdown option on gql
- [x] Fix export of gql request into HAR format
- [x] Fix issue on content-type header on GQL Request

### How did I test it ?

- I can export a GraphQL request with variables, and the exported code works (tested with bash and httpie)

image

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

File: packages/bruno-app/src/utils/codegenerator/har.js [hunk 1]
```diff
@@ -13,13 +13,22 @@ const createContentType = (mode) => {
...
-  return headers
+const createHeaders = (headers, type) => {
+  const result = headers
     .filter((header) => header.enabled)
     .map((header) => ({
       name: header.name,
       value: header.value
     }));
+
+  // TODO in this PR, make sure that the content type application/json is added to the headers by default
```
Comment (ajubin) [inline packages/bruno-app/src/utils/codegenerator/har.js]: I don't understand why GQL Request don't have the content-type "application/json" by default, Iguess this should be setup somewhere else in the code, as here it seems flaky

**Comment by dtrckd** [comment]
+1 for this PR, thank you.

**Comment by ajubin** [comment]
Cant code for the next month, feel free to merge it or modify it

**Comment by jeffabailey** [comment]
What happened to this? I can't use Bruno without this feature.

Is help needed? Is there another PR that provides this feature?

**Comment by crashingrob** [comment]
Bump, this is literally the only feature preventing me from switching to Bruno

**Review by sanjai0py — APPROVED** [review]

**Comment by helloanoop** [comment]
Merged!

**Pull request merged**